docs/requirements.md

```diff
@@ -31,40 +31,45 @@ previous one was not available.
-| `trace bind`             | 4.15 (BCC), 5.4 (CO:RE) |
-| `trace capabilities`     | 4.15                    |
-| `trace dns`              | 5.4                     |
-| `trace exec`             | 4.15 (BCC), 5.4 (CO:RE) |
-| `trace fsslower`         | 5.4                     |
-| `trace mount`            |                         |
-| `trace oomkill`          | 5.4                     |
-| `trace open`             | 4.15 (BCC), 5.4 (CO:RE) |
-| `trace signal`           | 5.4                     |
-| `trace sni`              |                         |
-| `trace tcp`              | 4.15                    |
-| `tracep tcpconnect`      | 4.15 (BCC), 5.8 (CO:RE) |
-| `traceloop`              | 4.15                    |
+| Gadget                   | Minimum Kernel          | Additional `CONFIG_*`   |
+|--------------------------|-------------------------| ----------------------- |
+| `advise network-policy`  | (BCC only)              |                         |
+| `advise seccomp-profile` | (CO-RE only)            | No specific ones        |
+| `audit seccomp`          | 5.4 (CO-RE only)        | `KPROBES`               |
+| `profile block-io`       | 4.15 (BCC only)         |                         |
+| `profile cpu`            | (BCC only)              |                         |
+| `snapshot process`       | 5.10 (CO-RE only)       | No specific ones        |
+| `snapshot socket`        | 5.10 (CO-RE only)       | No specific ones        |
+| `top block-io`           | (CO-RE only)            | `KPROBES`               |
+| `top file`               | 5.4 (CO-RE only)        | `KPROBES`               |
+| `top tcp`                | 4.15 (BCC), U.U (CO-RE) | `KPROBES`               |
+| `trace bind`             | 4.15 (BCC), 5.4 (CO-RE) | `KPROBES`, `KRETPROBES` |
+| `trace capabilities`     | 4.15 (BCC), U.U (CO-RE) | `KPROBES`               |
+| `trace dns`              | 5.4 (CO-RE only)        | No specific ones        |
+| `trace exec`             | 4.15 (BCC), 5.4 (CO-RE) | `FTRACE_SYSCALLS`       |
+| `trace fsslower`         | 5.4 (CO-RE only)        | `KPROBES`, `KRETPROBES` |
+| `trace mount`            | U.U (BCC), U.U (CO-RE)  | `FTRACE_SYSCALLS`       |
+| `trace oomkill`          | 5.4 (CO-RE only)        | `KPROBES`               |
+| `trace open`             | 4.15 (BCC), 5.4 (CO-RE) | `FTRACE_SYSCALLS`       |
+| `trace signal`           | 5.4 (CO-RE only)        | `FTRACE_SYSCALLS`       |
+| `trace sni`              | U.U (CO-RE only)        | No specific ones        |
+| `trace tcp`              | 4.15 (BCC only)         |                         |
+| `trace tcpconnect`       | 4.15 (BCC), 5.8 (CO-RE) | `KPROBES`, `KRETPROBES` |
+| `traceloop`              | 4.15 (CO-RE)            | `KPROBES`               |
+
+If the kernel version is U.U, it means we do not have this information at the
+moment.
+Also, "No specific ones" means that as long as you have
+`CONFIG_DEBUG_INFO_BTF=y`, the gadget should work.
```

Comment:
The `No specific ones` sentence is not clear IMO. What's about describing that all gadgets require `CONFIG_BPF=y` and `CONFIG_BPF_SYSCALL=y` and then that CO-RE ones require `CONFIG_DEBUG_INFO_BTF=y` when not supported by BTFHub?